tensorboard/webapp/webapp_data_source/BUILD

```diff
@@ -91,6 +91,7 @@ tf_ng_module(
 tf_ng_module(
     name = "feature_flag",
     srcs = [
+        "force_SVG_data_source.ts",
```

Comment:
Over time we've been trying to move things out of webapp_data_source and into feature-specific folders. Ideally force_svg_data_source and tb_feature_flag_data_source and tb_feature_flag_module would be in the feature_flag folder.

Could we put force_svg_data_source there immediately?

Would you be interested in following up with cleanup PRs to move tb_feature_flag_data_source and tb_feature_flag_module there as well? (disclosure: this may be a bit difficult because of the fact that classes in our internal repo reference these classes)

Reply:
Moved and renamed. Yea I can probably do that cleanup. I will look into what it will take to make that happen. Maybe make a copy of it temporarily until we can switch internal code over then remove.